cogboard-webapp/src/components/Widget/index.js

```diff
@@ -121,6 +121,8 @@ const Widget = ({ id, index }) => {
   return (
     <>
       <StyledCard
+        showShadow={expanded}
+        showBorder={isAuthenticated && 'WhiteSpaceWidget' === type}
```

Comment:
Wouldn't it be better to move `isAuthenticated && 'WhiteSpaceWidget' === type` to  variable and use it throughout component?

Reply:
done